### PR TITLE
Add IOPS/BPS to task size proto

### PR DESCRIPTION
**Pull request by bduffany**

### CODE REVIEW

File: proto/scheduler.proto [hunk 1]
```diff
@@ -107,6 +107,18 @@ message TaskSize {
...
+  int64 estimated_disk_wbps = 5;
+
+  // The task's estimated disk bytes read per second.
+  int64 estimated_disk_rbps = 6;
+
+  // The task's estimated disk read IO operations per second.
+  int64 estimated_disk_riops = 7;
+
+  // The task's estimated disk write IO operations per second.
+  int64 estimated_disk_wiops = 8;
```
Comment (tylerwilliams) [inline proto/scheduler.proto:120]: nit: all fields in here start with estimated_ which seems sort of redundant

**Review by tylerwilliams — APPROVED** [review]
LGTM with nits

File: proto/scheduler.proto (hunk 1 above)
Comment (bduffany) [reply proto/scheduler.proto]: yeah, it's a pre-existing issue, but I got rid of `estimated_` for now and we can rename the other fields later (should be wire-compatible)

[the remark this answers is not in the document]
Comment (bduffany) [reply proto/scheduler.proto]: good call, done

**Pull request merged**